packages/core/src/codewhisperer/util/authUtil.ts

```diff
@@ -345,6 +348,30 @@ export class AuthUtil {
         }
     }
 
+    public async notifySessionConfiguration() {
+        const suppressId = 'amazonQSessionConfigurationMessage'
+        const settings = AmazonQPromptSettings.instance
+        const shouldShow = await settings.isPromptEnabled(suppressId)
+        if (!shouldShow) {
+            return
+        }
+
+        const message = localize(
+            'aws.amazonq.sessionConfiguration.message',
+            'Your maximum session length for Amazon Q can be extended to 90 days by your administrator. For more information, refer to How to extend the session duration for Amazon Q in the IDE in the IAM Identity Center User Guide.'
+        )
+
+        const learnMoreUrl = vscode.Uri.parse(
+            'https://docs.aws.amazon.com/singlesignon/latest/userguide/configure-user-session.html#90-day-extended-session-duration'
+        )
+        await vscode.window.showInformationMessage(message, localizedText.learnMore).then(async (resp) => {
```

Comment:
```suggestion
        void vscode.window.showInformationMessage(message, localizedText.learnMore).then(async (resp) => {
```

Q activation will be blocked on the user interacting with this message (close, click a button). If they just ignore it or minimize the notifications then this await will never resolve.